cdk-infra/lib/constructs/worker/worker-construct.ts

```diff
@@ -89,9 +89,22 @@ export class WorkerConstruct extends Construct {
       executionRole,
     });
 
+    const updateTaskProtectionPolicy = new PolicyStatement({
```

Comment:
grants the task the ability to protect itself from being stopped during a deployment or scale-in